import.py

```diff
@@ -672,7 +674,7 @@ def get_decompme_compiler_name(
     trail = "permuter_settings.toml, where ... is one of: " + ", ".join(available_ids)
     if compiler_mappings:
         print(
-            "Please add an entry:\n\n"
+            "Please add an entry: (wildcards allowed!)\n\n"
```

Comment:
Add it to the print below as well, for the no-pre-existing section case. (I'd make a review suggestion and accept it myself but github only allows making suggestions up to three lines away from the modified code...) Also, maybe move it to the end instead of here after the colon which looks a bit weird to me grammatically?